src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Input/TextCompositionManager.cs

```diff
@@ -23,6 +23,7 @@
 using MS.Internal.PresentationCore;                        // SecurityHelper
 
 using SR=MS.Internal.PresentationCore.SR;
+using UnsafeNativeMethods = MS.Win32.UnsafeNativeMethods;
```

Comment:
These are unfortunate, and if absolutely needed to disabiguate - we can definite it central as a [global using](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#the-global-modifier) for .NET Framework builds.

Reply:
I did as many central global usings as I could, but some still needed direct usings due to conflicts or how the namespaces were declared in the file.